Pat.Subscriber.IntegrationTests/SubscriberFixture.cs

```diff
@@ -72,8 +72,15 @@ public SubscriberFixture()
 
```

Comment:
Hello mate, I've made a couple of general comments.
When I get a min I'm gonna pull down the code and take a proper look as I'm not familiar enough with the rule generation.

Cheers,
Steve